### ISSUE TITLE
[1.12.2 pre118] Advanced Recipe scaling issue

### ISSUE THREAD

**Issue by kirokadura** [CreativeMD/LittleTiles#134]
Reported by user *_bildo_* on Discord.

LittleTiles - v1.5.0 pre118 
CreativeCore - v1.9.35 
Minecraft / Forge - v1.12.2 / v14.23.5.2768
Java settings - Default, except 8G RAM allocated instead of 1G

EDIT: DISREGARD ~~(The Advanced Recipe can't scale up anything if the result is bigger than a block and can't scale down if the source block is bigger than a block.)~~

EDIT: CORRECT DESCRIPTION : The Advanced Recipe can't scale down anything if the result is bigger than a block and can't scale up if the source block is bigger than a block.

A example is provided below. You can't scale it down, unless the 32 grid option is selected, presumably because it's smaller than a block.

>{tiles:[{boxes:[[I;0,0,0,1,1,1],[I;0,1,0,1,2,1],[I;0,2,0,1,3,1],[I;0,3,0,1,4,1],[I;0,4,0,1,5,1],[I;0,5,0,1,6,1],[I;0,6,0,1,7,1],[I;0,7,0,1,8,1],[I;0,8,0,1,9,1],[I;0,9,0,1,10,1],[I;0,10,0,1,11,1],[I;0,11,0,1,12,1],[I;0,12,0,1,13,1],[I;0,13,0,1,14,1],[I;0,14,0,1,15,1],[I;0,15,0,1,16,1],[I;0,16,0,1,17,1]],tile:{meta:0,block:"minecraft:dirt",tID:"BlockTileBlock"},group:1b}],min:[I;0,0,0],size:[I;1,17,1],grid:1,count:17}

**Comment by CreativeMD** [comment]
I know about the scale up thingy. The idea was to prevent people from taking vanilla stuff and making it even bigger. That's not really what LittleTiles is about.
Scaling down on the other hand should always work, but keep in mind if you have selected a 32 grid, you cannot scale it down, because you already reached the limit, you would need 64 grid for that. I could scale down the provided structure without any problem.

**Comment by CreativeMD** [comment]
I fixed a minor bug about scaling down children. Tested it again, the same way you showed me and it worked, so I guess it's fixed now. I will upload the new version soon, I would be really happy if you could test it again.

**Comment by kirokadura** [comment]
Just tested it and the issue persisted. If I select the dirt tower with the Advanced Recipe I can save it as is, but I can't save it on 1:2, 1:4, 1:8 and 1:16 settings. 1:32 beyond works. 

Same thing about the scaled down version. It refuses to scale up to any size if the original structure is taller/bigger than a block. So if I want to scale it to the 1:16, I'll have to scale it down to 1:32 then scale it up to 32:16.

As an example, the picture below is of a model I made of a village (pictured in the back). I miniaturized it using the old Miniature Recipe. I can no longer shrink it with the Advanced Recipe because clicking Save does nothing. I tested it up until grid 64 and had no success.
![2018-10-26_18 40 54](https://user-images.githubusercontent.com/29471180/47591525-54e48280-d946-11e8-9fbb-de8280f7df73.png)

I tested it with only a small portion of the village and it allowed me to Save with the 64 grid option, because the result was smaller than a block.

![2018-10-26_18 44 16](https://user-images.githubusercontent.com/29471180/47591689-e18f4080-d946-11e8-9166-09d19f8328be.png)
